README.md

```diff
@@ -101,11 +101,12 @@ This means that the CSV may contain only those columns that contain changed valu
 
 #### Base attributes
 
-To create or update products you need 2 columns.
-You always need the `productType`. Further you need either `variantId` or `sku` to identify the specific variant.
+To create or update products you need to provide the following headers and columns. 
+You will always need the `productType` on the master variant. Further you need to provide the `variantId` and `sku` to define the specific variants.
```

Comment:
@ebonertz based on the description of this PR it sounds like `variantId` is mandatory, but `sku` is optional. The way you phrase it here though tells me that `sku` is mandatory too. Is this what you would like to say?